test/test_helper.rb

```diff
@@ -20,26 +20,41 @@ def analyze_theme(*check_classes, templates)
     end
 
     def make_theme(files = {})
+      # make_file_system_theme(files)
+      make_in_memory_theme(files)
```

Comment:
One of those two is commented out because I was running the tests with both to make sure everything passes regardless of which kind of storage you're using. Would there be a better way to make it so all the make_theme tests run on both?

Reply:
The only way I can think of is to use an env var: `TEST_STORAGE=file_system bundle exec rake`, then check `ENV["TEST_STORAGE"]` here.

You could create a rake task to run each one, and run both by default.

Reply:
Why, of course! Nice!